### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by DaftonJ** [peripheryapp/periphery#340] [Q&A]
Hi.
I have some problem with Periphery. When I trying to use command `periphery scan --setup i`n terminal, I have only sth like this `Shell command '/usr/bin/env xcodebuild -workspace /Users/jskd/Desktop/app/App.xcworkspace -list -json' returned exit status '1'`.
Maybe u know where is the problem?

**Comment by ileitch** [comment]
Is there no other output? Can you try running `/usr/bin/env xcodebuild -workspace /Users/jskd/Desktop/app/App.xcworkspace -list -json` in your project directory and copy any output here.

**Reply by DaftonJ** [reply]
Here is output `xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance`

**Reply by ileitch** [reply]
You need to setup Xcode for command line, use: `sudo xcode-select -s /Applications/Xcode.app`, change the path to Xcode.app if it's different on your system.

**Reply by DaftonJ** [reply]
It works, Thanks !
